handbook/people-ops/compensation/equity-faq.md

```diff
@@ -0,0 +1,69 @@
+
+### So, exactly how valuable are my options today?
+
+Since Sourcegraph’s stock isn’t actively or publicly traded, there is unfortunately no way to know what they are worth "today”. It is possible to use the _latest_ fundraising valuation to calculate what they could have been worth if you had exercised them and could have sold them in a past fundraising round. We provide this calculation when presenting offers.
+
+However, that is a flawed view. Because our stock _can’t_ be sold today, we have to make predictions about what it could be worth in the future. This is why we present a range of outcomes in offer presentations instead of just using the latest value (but of course, even using scenarios is imprecise, due to unknowns like future dilution and tax rates).
+
+Importantly, because Sourcegraph is a growing startup, looking at the latest value in our offer presentation leaves out a lot of important nuance. You could go work at a larger, more established company, and get an equity grant worth more "today", but the upside may be lower (and of course the downside may also be lesser). Volatility, or the expectation of change in the future, is fundamentally valuable when it comes to options.
+
+So while it’s easy to plug our recent numbers into one of several calculators, it’s just not that simple. We believe that a view of different future scenarios like we provide in offer letters is the best way to communicate a range of outcomes. The People Ops team can provide this calculation for you if you are ever interested.
+
+### How are option grants calculated for new hires?
+
+See our [compensation bands overview](index.md#how-bands-are-created) to understand how option grants for new hires are calculated.
+
+### How are option grants calculated for promotions?
+
+As a company grows in size, the value of each share (and option) grows, which means that the _number of shares_ granted will decline over time.
+
+This means that simply joining the company earlier can result in someone having more options than the latest bands would suggest. And if they are promoted, they could already have more options than the new, theoretically higher band would reflect.
+
+In such cases, teammates are not intended to be punished for joining earlier. If necessary, they will generally receive an additional number of options that matches the _increase_ between their previous and new levels in the latest bands. Some variation can exist in certain situations.
+
+As an example of how this could work in practice, imagine a teammate already has 1,000 options. The company's bands have evolved since that iniital grant, and they are later promoted from their current level, which has a new target of 300 options, to a higher level that has a target of 425 options. For their promotion, this individual would receive +125 options (the difference between 425 and 300) to get them to 1,125 total.
```

Comment:
```suggestion
As an example of how this could work in practice, imagine a teammate already has 1,000 options. The company's bands have evolved since that initial grant, and they are later promoted from their current level, which has a new target of 300 options, to a higher level that has a target of 425 options. For their promotion, this individual would receive +125 options (the difference between 425 and 300) to get them to 1,125 total.
```